File: scripts/Two Sum II - Input Array Is Sorted/Two Sum II - Input Array Is Sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        s=0
        e=len(numbers)-1
        while s<e:
            sumi = numbers[s]+numbers[e]
            if(sumi==target):
                return [s+1, e+1]
                
            elif(sumi<target):
                s+=1
            
            else:
                e-=1

File: scripts/algorithms/T/Two Sum II - Input Array Is Sorted/Two Sum II - Input Array Is Sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int low = 0;
        int high = numbers.size()-1; 
        vector<int> ans;
        while(low<=high)
        {
            int sum =numbers[low] + numbers[high] ; 
            
            if(sum == target)    
                return vector<int>{ low+1, high+1};
            
            else if(sum>target)
                high=high-1;
            
            else
                low=low+1;
        }
         return vector<int>{ };;
    }
    
};
